docs/concept/how-to-use-predefined-hourly-setpoints.rst

```diff
@@ -0,0 +1,41 @@
+:orphan:
+
+:Author: Sreepathi Bhargava Krishna
+:Date: 26-February-2019
+
+How to calculate demand using prefined-hourly-setpoints in CEA?
+===============================================================
+
+Step 1: In the `config` file, make the `demand.predefined-hourly-setpoints` as `True`
+
+Step 2: Create `csv` files corresponding to `space-heating` and/or `space-cooling`. The files are to be named in the
+`BuildingName_temperature.csv` format. The file content should be the values of time and the corresponding set point
+temperature in the columns: `time` and `temperature`
```

Comment:
- this would be a good place for an "example" of the file... maybe just the first 5 rows including the header. 
- also, this documentation does not explain where to put the files. Please mention the path
- the phrase "the values of time" is a bit vague. Do you mean date + time values for each hour of the year? What is the format used to encode the date? your example in the PR looks like "dd.mm.yyyy", but e.g. in america "mm.dd.yyyy" is also common - how does the CEA deal with this? https://en.wikipedia.org/wiki/ISO_8601 goes into this a bit, but you might be piggy backing off of pandas here? if so, mention that, so a user can go check there instead.